src/jmcomic/jm_config.py

```
@@ -551,3 +551,55 @@ def register_exception_listener(cls, etype, listener):
+        formatted = super().format(record)
+        return f'{color}{formatted}{self.RESET}' if color else formatted
+
+
+def enable_pretty_log():
+    """开启带颜色的美化日志"""
+    import sys
+    import os as _os
+
+    # Windows 需要启用 VT100 ANSI 支持
+    if sys.platform == 'win32':
+        _os.system('')
```

Ruff (0.15.12)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)